### PR TITLE
chore: scaffolding for enabling code formatting

### PR DESCRIPTION
**Pull request by Andreagit97**
**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

/kind cleanup

/kind feature

**Any specific area of the project related to this PR?**

/area build

/area CI

**What this PR does / why we need it**:

This is the dual of https://github.com/falcosecurity/libs/pull/2038 for libs if we want to apply the format also here! It would be great to do it before detaching the release branch

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: .clang-format-ignore
Comment (Andreagit97) [inline .clang-format-ignore:1]: Do we have some files to add here? @FedeDP

**Comment by Andreagit97** [comment]
@falcosecurity/falco-maintainers let me know what you think about this

File: .clang-format-ignore
Comment (FedeDP) [reply .clang-format-ignore:1]: None that i know of :D

**Comment by FedeDP** [comment]
+1 from me!
cc @alacuku that is the release manager though.

**Comment by leogr** [comment]
+1 from me too!

**Comment by LucaGuerra** [comment]
+1

**Comment by alacuku** [comment]
If the maintainers are all on board, then it is a +1 from me too.

**Review by leogr — APPROVED** [review]

**Review by FedeDP — APPROVED** [review]
/approve

**Pull request merged**